test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -25,12 +25,13 @@ notifyFormat_ngsi10
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB 0 IPv4 -mutexTimeStat
+brokerStart CB 0 IPv4 -mutexTimeStat -subCacheIval 0
```

Comment:
I have checked that it is safe to remove `-subCacheIval 0`, running the test under valgrind three times:

```
[develenv@ci-fiware-01 fiware-orion]$ test/valgrind/valgrindTestSuite.sh -filter three_subscriptions.test
Thu Sep 10 13:51:05 CEST 2015
Test 001/1: 1048_subscription_cache/three_subscriptions ............................................................................................. OK (13.82 seconds)
Total test time: 13.94 seconds (0 minutes)
Great, all valgrind tests ran without any memory leakage
[develenv@ci-fiware-01 fiware-orion]$ test/valgrind/valgrindTestSuite.sh -filter three_subscriptions.test
Thu Sep 10 13:51:22 CEST 2015
Test 001/1: 1048_subscription_cache/three_subscriptions ............................................................................................. OK (13.34 seconds)
Total test time: 13.46 seconds (0 minutes)
Great, all valgrind tests ran without any memory leakage
[develenv@ci-fiware-01 fiware-orion]$ test/valgrind/valgrindTestSuite.sh -filter three_subscriptions.test
Thu Sep 10 13:51:37 CEST 2015
Test 001/1: 1048_subscription_cache/three_subscriptions ............................................................................................. OK (13.34 seconds)
Total test time: 13.47 seconds (0 minutes)
Great, all valgrind tests ran without any memory leakage
[develenv@ci-fiware-01 fiware-orion]$ test/valgrind/valgrindTestSuite.sh -filter three_subscriptions.test
Thu Sep 10 13:51:54 CEST 2015
Test 001/1: 1048_subscription_cache/three_subscriptions ............................................................................................. OK (13.38 seconds)
Total test time: 13.51 seconds (0 minutes)
Great, all valgrind tests ran without any memory leakage
```